lib/yoti/data_type/document_details.rb

```diff
@@ -51,28 +51,20 @@ class DocumentDetails
     # @param [String] value
     #
     def initialize(value)
-      validate_value(value)
       parse_value(value)
     end
 
     private
 
-    #
-    # Asserts provided matches VALIDATION_PATTERN
-    #
-    # @param [String] value
-    #
-    def validate_value(value)
-      raise(ArgumentError, "Invalid value for #{self.class.name}") unless /#{VALIDATION_PATTERN}/.match?(value)
-    end
-
     #
     # Parses provided value into separate attributes
     #
     # @param [String] value
     #
     def parse_value(value)
-      attributes = value.split(' ')
+      attributes = value.split(/\s/)
```

Comment:
Do we really want to split on any whitespace character? I'd assume this'd be a breaking change from Connect if it started passing us back tab-separated values?

Reply:
> Do we really want to split on any whitespace character? I'd assume this'd be a breaking change from Connect if it started passing us back tab-separated values?
> 
> Alternatively do we want to change the other SDKs to match any ws?

Reply:
Fixed in f26aefe